src/_nebari/stages/kubernetes_services/__init__.py

```diff
@@ -61,7 +61,10 @@ class DefaultImages(schema.Base):
 
 
 class Storage(schema.Base):
-    type: SharedFsEnum = None
+    type: SharedFsEnum = Field(
+        default=None,
+        json_schema_extra=schema.ExtraFieldSchema(immutable=True).model_dump(),
```

Comment:
Curious if there are any thoughts on this.  Creating ExtraFieldSchema may be overkill since
`json_schema_extra={'immutable': True)` would work just as well.

Reply:
@viniciusdc

Reply:
sorry @Adam-D-Lewis  missed your ping here, I know its kind late but I also agree with you. Also, nice found for the immutable thing, really helped me out on another PR too